v1/examples/basics/pages/en/index.js

```diff
@@ -20,7 +20,9 @@ function imgUrl(img) {
 }
 
 function docUrl(doc, language) {
-  return `${siteConfig.baseUrl}docs/${language ? `${language}/` : ''}${doc}`;
+  return `${siteConfig.baseUrl}${siteConfig.docsUrl}/${
```

Comment:
Does this work even when docsUrl is null ? In which it should have fallbacked to docs